doc/UserGuide.md

```diff
@@ -39,15 +39,15 @@ Format: `add NAME [p]p/PHONE_NUMBER [p]e/EMAIL [p]a/ADDRESS [t/TAG]...`
  
 > Words in `UPPER_CASE` are the parameters, items in `SQUARE_BRACKETS` are optional, 
 > items with `...` after them can have multiple instances. Order of parameters are fixed. 
-> 
+>
 > Put a `p` before the phone / email / address prefixes to mark it as `private`. `private` details can only
 > be seen using the `viewall` command.
 > 
 > Persons can have any number of tags (including 0)
 
 Examples: 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe pp/1234567 e/betsycrowe@gmail.com pa/Newgate Prison t/criminal t/friend`
+* `add John Doe p/98765432 e/johnd@gmail.com a/Block 123, John Street, #01-01, 600000`
+* `add Betsy Crowe pp/1234567 e/betsycrowe@gmail.com pa/Block 1, Newgate Prison, 1, 500000 t/criminal t/friend`
```

Comment:
It would help if you added the correct format in the user guide. How would the user know the exact format otherwise?